docs/community-events/20240221-Intro-to-TROLIE.md

```diff
@@ -0,0 +1,153 @@
+be using the `resource-id` field to identify a branch when branches are not
+assigned a single identifier?</h3>
+
+We created a new issue for this: [#54](https://github.com/trolie/spec/issues/54).
+Please join the discussion there.
+
+<h3> How are Monitoring Sets, Transmission Facilities and Segments defined and
+updated by Ratings Provider? What are the validation rules? How are they
+coordinated with network model updates</h3>
+
+TROLIE is intentionally agnostic to how server and client implementations update 
+their network models, or whether these data entities are part of the network model 
+at all, or updated via some other means.  The validation rules provided are also 
+up to individual implementations.  
+
+We understand that this is may be disappointing, because maintenance of 
+network model databases and orchestrating their roll out can be
+challenging, especially across entities.  
+
+Exchanging model data is _not_ in the scope of TROLIE.  This is an incredibly
+complex problem that likely requires its own set of defined protocols, standards, 
+discussions and vendor adoption, much like the 
+Common Grid Model Exchange Standard (CGMES) used in the 
+ENTSO-E (https://www.entsoe.eu/data/cim/cim-for-grid-models-exchange/). 
+
+As of this writing, the processes to update, exchange and orchestrate network model
+updates in North America are highly fragmented.  TROLIE does not attempt to solve this problem.  
+
+However, instead, we recognize this fragmented landscape as a reality that users of TROLIE must 
+navigate.  In the short term, please consult appropriate vendors, reliability coordinators
+or partners on how their specific software behaves.  In addition, we have created issue
+https://github.com/trolie/spec/issues/57 to document recommended validation rules and model
+coordination best-practices for TROLIE implementers.  
+
+### Would TROLIE be a cloud-based solution?
+
+TROLIE is agnostic to where servers are hosted.  The specification will work whether clients or
+servers are hosted either on-premise or in a public cloud.  
+
+Specific vendor products and reliability coordinator implementations will of course be more opinionated.  
+
+### Does the specification allow for UTC timestamps?
+
+Yes.  TROLIE timestamps use the RFC 3339 format (https://www.rfc-editor.org/rfc/rfc3339).  
+This format is daylight savings-safe, as shown by example at https://trolie.energy/daylight-savings.  
+
+The example referenced above however shows timestamps in local time zones.  The common way to specify UTC is
+to use a "Z" character instead of a UTC offset as a suffix, like in the following example, which evaluates to 7am EDT:
+
+    2025-11-01T11:00:00Z
+
+
+<h3>How are the status/state of Real-Time and Forecast snapshots communicated
+via TROLIE interfaces? How can we know from the TROLIE interface that a new
+update for the snapshot values is available?</h3>
+
+including the restriction of the TROLIE Specification to classic REST patterns
```

Comment:
Looks like you're missing the first part of your sentence here.  It reads pretty strange.